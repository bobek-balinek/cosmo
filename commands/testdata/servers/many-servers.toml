[servers.server-1]
host = 'host-1'
user = 'user-1'

  [servers.server-1.tasks.task-1]
    remote = [
      { exec = 'echo task-1' },
    ]

  [servers.server-1.tasks.task-2]
    remote = [
      { exec = 'echo task-2' },
    ]

[servers.server-2]
host = 'longhost-2'
user = 'longuser-2'

  [servers.server-2.tasks.task-1]
    remote = [
      { exec = 'echo task-1' },
    ]

[servers.server-3]
host = 'host-3'
user = 'user-3'

  [servers.server-3.tasks.task-1]
    remote = [
      { exec = 'echo task-1' },
    ]
