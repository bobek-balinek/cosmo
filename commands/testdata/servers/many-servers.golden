server-1  user-1@host-1          tasks: 2
server-2  longuser-2@longhost-2  tasks: 1
server-3  user-3@host-3          tasks: 1
